gobblin-api/src/main/java/org/apache/gobblin/configuration/ConfigurationKeys.java

```diff
@@ -833,6 +835,7 @@
   public static final String KAFKA_SCHEMA_REGISTRY_RETRY_TIMES = "kafka.schema.registry.retry.times";
   public static final String KAFKA_SCHEMA_REGISTRY_RETRY_INTERVAL_IN_MILLIS =
       "kafka.schema.registry.retry.interval.inMillis";
+  public static final String SCHEMA_CREATION_TIME_KEY = "CreatedOn";
```

Comment:
If this is supposed to be an internal attribute, you won't need it as public static but should be as close as possible to wherever you are using it. ( I am a bit opposed to adding anything more into ConfigurationKeys but looks like it is common practice that everyone is following.

Reply:
Will address